### PR TITLE
Update Add-SPOHubSiteAssociation.md with multi-geo information

### PR DESCRIPTION
**Pull request by JaredRic-corp**
Added a note clarifying the values necessary when assigning a hubsite across geo locations.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOHubSiteAssociation.md [hunk 1]
```diff
@@ -81,3 +81,5 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 If the site or hub site doesn't exist, this cmdlet returns a "File not found" error.
 
 If the site is already a hub site, this cmdlet returns a "This site is already a HubSite" error.
+
+In multi-geo situations, when assigning a hub that is across geo locations you must pass the site ID of the hub site to the HubSite parameter as a URL will fail.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOHubSiteAssociation.md:85]:
```suggestion
In multi-geo situations, when assigning a hub that is across geo locations, you must pass the site ID of the hub site to the HubSite parameter as a URL will fail.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JaredRic-corp, kindly commit the minor suggested change. Many thanks.

**Review by JaredRic-corp — COMMENTED** [review]
Committed grammar change and updated Hubsite parameter description to include Site ID as well.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward Please review. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**